### DISCUSSION TITLE
Welcome to podgrab Discussions!

### DISCUSSION THREAD

**Discussion by akhilrex** [akhilrex/podgrab#208] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akhilrex** [comment]
Sorry for being unresponsive for the past few months. I plan to get into the issues now

Starting with #203

It seems podcasts with super long names are failing because most OS's have a defined max path length. I could find some windows level details [here ](https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file?redirectedfrom=MSDN ). Pretty sure something like this would exist for linux also.

Now there are 2 options that come to my mind
1. Truncation of the title
2. Renaming to a random

both methods have their own inherent issues. I just wanted to get a sense of what the community feels would be the right way.

**Reply by SolveSoul** [reply]
If ID3 tags are going to be supported in the future then the filename might be renamed based on the tags similar to .mp3 tracks for albums (artist - title.mp3) or any other variant.

But even then there might be a path length issue so I'd vote for truncating the title.
